src/DecoratorManifold.jl

```diff
@@ -268,16 +268,18 @@ macro decorator_transparent_function(fallback_case, input_ex)
                 $(callargs[2:end]...);
                 $(kwargs_list...),
             ) where {$(where_exprs...)}
-                return error(string(
-                    ManifoldsBase.manifold_function_not_implemented_message(
-                        $(argnames[1]),
-                        $fname,
-                        $(argnames[2:end]...),
+                return error(
+                    string(
+                        ManifoldsBase.manifold_function_not_implemented_message(
+                            $(argnames[1]),
+                            $fname,
+                            $(argnames[2:end]...),
+                        ),
+                        " Usually this is implemented for a ",
+                        $(argtypes[1]),
+                        ". Maybe you missed to implement this function for a default?",
```

Comment:
“this function for a default” – you are referring to a default type? Otherwise one could also write „a default for this function“.

Reply:
This text was there before. Now I'm not quite sure what was it supposed to say.

Reply:
Ah, I did not notice that; we could still rephrase it :)